pytradfri/api/libcoap_api.py

```diff
@@ -103,10 +103,13 @@ def _observe(self, api_command):
         """Observe an endpoint."""
         path = api_command.path
         duration = api_command.observe_duration
+        if duration <= 0:
+            duration = 1
```

Comment:
Throw a ValueError if it's out of bounds.

Reply:
I'm hesitant to do that since the default value of duration is 0 in `command.py`. At the same time, I'm hesitant to change that default.

Reply:
After digging a little deeper I found that default observation duration is 60, not 0. I'll throw the ValueError instead.